cmd/node/config/config.toml

```diff
@@ -12,42 +12,42 @@
    StartInEpochEnabled = true
 
    # SCDeployEnableEpoch represents the epoch when the deployment of smart contracts will be enabled
-   SCDeployEnableEpoch = 2
+   SCDeployEnableEpoch = 1
```

Comment:
I would revert (value 2) the following flags: `SCDeployEnableEpoch`, `BuiltInFunctionsEnableEpoch`, `RelayedTransactionsEnableEpoch`, `TransactionSignedWithTxHashEnableEpoch`, `MetaProtectionEnableEpoch`,  `AheadOfTimeGasUsageEnableEpoch`, `EpochEnable for the second configuration`

Reply:
done